src/main/java/kitchenpos/order/application/OrderService.java

```diff
@@ -0,0 +1,45 @@
+package kitchenpos.order.application;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderRepository;
+import kitchenpos.order.domain.OrderStatus;
+import kitchenpos.order.dto.OrderRequest;
+import kitchenpos.order.dto.OrderResponse;
+import kitchenpos.order.event.CreateOrderEvent;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class OrderService {
+    private final OrderRepository orderRepository;
+    private final ApplicationEventPublisher eventPublisher;
+
+    public OrderService(OrderRepository orderRepository, ApplicationEventPublisher eventPublisher) {
+        this.orderRepository = orderRepository;
+        this.eventPublisher = eventPublisher;
+    }
+
+    @Transactional
+    public OrderResponse create(final OrderRequest request) {
+        final Order savedOrder = orderRepository.save(request.toOrder());
+        eventPublisher.publishEvent(new CreateOrderEvent(request.getOrderTableId(), request.getMenuIds()));
```

Comment:
```suggestion
        eventPublisher.publishEvent(new OrderCreatedEvent(request.getOrderTableId(), request.getMenuIds()));
```

주문이 생성되어 발생했다라는 사실을 네이밍을 통해 명확하게 남겨줄 수 있을 것 같습니다!

Reply:
이 리뷰를 놓쳤네요 수정하겠습니다!